### PR TITLE
net/ble: cleanup doxygen grouping

### PR DESCRIPTION
**Pull request by haukepetersen**
### Contribution description
This PR adds some more structure to the doxygen grouping of the 'BLE subsystem'. Now all BLE groups are subgroups of a global `ble` (EDIT) doxygen group.

### Testing procedure
Compile a verify doxygen html output

### Issues/PRs references
none

### CODE REVIEW

**Comment by haukepetersen** [comment]
fixed a minor issue doxygen pointed out (squashed right in)

File: pkg/nimble/doc.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /**
- * @defgroup pkg_nimble     NimBLE BLE stack
+ * @defgroup ble_nimble NimBLE
  * @ingroup  pkg
- * @ingroup  net
+ * @ingroup  net_ble
```
Comment (miri64) [inline pkg/nimble/doc.txt]: `ble`

Comment (miri64) [reply pkg/nimble/doc.txt]: Then again, why not prefix everything with `net_` to stick to the common theme?

Comment (haukepetersen) [reply pkg/nimble/doc.txt]: ups

**Comment by haukepetersen** [comment]
fixed nimble groups (hopefully now murdock will be happy...)

**Comment by haukepetersen** [comment]
Murdock is happy, @miri64 are you too?

**Pull request merged**